File: C#/프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.cs
using System;

public class Solution {
    public string solution(string my_string, string overwrite_string, int s) {
        int lo = overwrite_string.Length;
        string answer = my_string.Substring(0,s)+overwrite_string+my_string.Substring(s+lo);
        return answer;
    }
}

File: Python3/프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.py
def solution(my_string, overwrite_string, s):
    answer = ''
    for i in range(0,len(my_string)):
        if i>=s and i<s+len(overwrite_string):
            answer+= overwrite_string[i-s]
        else:
            answer+=my_string[i]
    return answer